Sources/Sentry/include/SentryLog.h

```diff
@@ -11,6 +11,10 @@ SENTRY_NO_INIT
 
 + (void)logWithMessage:(NSString *)message andLevel:(SentryLevel)level;
 
+/** @return @c YES if the current logging configuration will log statements at the current level, @c
+ * NO if not. */
++ (BOOL)willLogAtLevel:(SentryLevel)level;
```

Comment:
`l`: `logWithMessage` could also return a `BOOL` if it logged the message.

Reply:
I did this as a sort of stopgap to what I'd really want, which is a macro that gets compiled out of runtimes when it's below the configured log level, (although we may not want that because then it could never be dynamically changed at runtime if we want to debug something). I'm using this to avoid doing the calculations to log debug statements in `logSlicingFailureWithArray`.